.github/workflows/bench.yml

```diff
@@ -0,0 +1,52 @@
+name: 'Benchmark'
+
+on:
+  pull_request:
+
+jobs:
+  self:
+    name: Georgia Tech | Phoenix (NVHPC)
+    if: github.repository == 'MFlowCode/MFC'
+    strategy:
+      matrix:
+        device: ['cpu', 'gpu']
+    runs-on:
+      group:  phoenix
+      labels: self-hosted
+    steps:
+      - name: Clone - PR
+        uses: actions/checkout@v3
+
+      - name: Bench - PR
+        run: |
+          bash .github/workflows/phoenix/submit.sh .github/workflows/phoenix/bench.sh ${{ matrix.device }}
+          mv bench-${{ matrix.device }}.out ~/bench-${{ matrix.device }}-pr.out
+          mv bench-${{ matrix.device }}.yaml ~/bench-${{ matrix.device }}-pr.yaml
+
+      - name: Clone - Master
+        uses: actions/checkout@v3
+        with:
+          repository: henryleberre/MFC
+          ref: master
+
+      - name: Bench - Master
+        run: |
+          bash .github/workflows/phoenix/submit.sh .github/workflows/phoenix/bench.sh ${{ matrix.device }}
+          mv bench-${{ matrix.device }}.out ~/bench-${{ matrix.device }}-master.out
```

Comment:
The home directory here is my user-space home directory on Phoenix...

Reply:
Addressed!